### DISCUSSION TITLE
Scanco file reader

### DISCUSSION THREAD

**Discussion by tknopp** [JuliaImageRecon/Sinograms.jl#45] [Show and tell]
Hey Jeff,

I moved my local code for reading Scanco microCT files to https://github.com/JuliaImageRecon/ScancoFiles.jl. Although the code is work in progress, it is already more generic in some aspects than https://github.com/KitwareMedical/ITKIOScanco, since I can read out the entire geometry information from the file itself.

Questions:
* Should we use vendor specific packages (ScancoFiles.jl) or do you prefer a larger package like "CTFiles.jl". In MRI I used the "MRIFiles.jl" approach but I am not sure if that was the best idea. Vendor specific packages seem to be also fine.
* I put it into JuliaImageRecon but we could also move it to JuliaIO. There is no consensus if such packages should live in JuliaIO or in domain specific organizations (see https://discourse.julialang.org/t/move-asdf-jl-to-juliaio/21637 for discussions). I don't have any preference there. However, I don't have admin rights for JuliaIO right now.

Thanks for your thoughts!

Tobi

**Comment by JeffFessler** [comment]
Vendor-specific packages are fine and happy to have it located here to possibly increase traffic for CT recon.
One thing I will be curious about is whether `CtGeom` here in Sinograms.jl is general enough, or whether additional geometry features are needed to support that scanner.  Let me know.
